step-function/get-files/src/main.py

```diff
@@ -27,34 +27,50 @@ def get_object_file_type(bucket, key):
     return None
 
 
-def lambda_handler(event, context):
-    bucket = environ['BUCKET']
-    response = S3_CLIENT.list_objects_v2(Bucket=bucket, Prefix=event['job_id'])
+def get_products(files):
+    return [{
+        'url': item['download_url'],
+        'size': item['size'],
+        'filename': item['filename']
+    } for item in files if item['file_type'] == 'product']
+
+
+def get_browse(files):
+    browse = [item for item in files if item['file_type'] in ('amp_browse', 'rgb_browse')]
+    sorted_files = sorted(browse, key=lambda x: x['file_type'])
+    urls = [item['download_url'] for item in sorted_files]
+    return urls
+
+
+def get_thumbnail(files):
+    return [item['download_url'] for item in files if item['file_type'] in ('amp_thumbnail', 'rgb_thumbnail')]
```

Comment:
`if 'thumbnail' in item['file_type']`
we should probably sort these, too, if we're going to keep both of them